component/async/kafka/simple/simple.go

```diff
@@ -34,6 +35,18 @@ func WithDurationOffset(since time.Duration, timeExtractor TimeExtractor) kafka.
 	}
 }
 
+// WithNotificationOnceReachingLatestOffset closes the input channel once all the partition consumers have reached the
+// latest offset.
+func WithNotificationOnceReachingLatestOffset(ch chan<- struct{}) kafka.OptionFunc {
```

Comment:
Is this a feature that makes sense only for the simple async Kafka component or possibly could be implemented also for the other one?

Reply:
I'd say it's less likely as (at least for us), when we want to use this feature is for stateful service that needs to consume events from a given topic (either log compacted or since a certain time). We don't want to listen to only 1 out of x partitions solely.